client/my-sites/site-settings/manage-connection/public-post-types.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { flowRight, pick } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import CompactFormToggle from 'components/forms/form-toggle/compact';
+import config from 'config';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import wrapSettingsForm from 'my-sites/site-settings/wrap-settings-form';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { isJetpackMinimumVersion, isJetpackSite } from 'state/sites/selectors';
+
+const PublicPostTypes = ( {
+	fields,
+	handleAutosavingToggle,
+	isRequestingSettings,
+	isSavingSettings,
+	supportsPublicPostTypesCheckbox,
+	translate
+} ) => {
+	if ( ! config.isEnabled( 'manage/option_sync_non_public_post_stati' ) || ! supportsPublicPostTypesCheckbox ) {
```

Comment:
I note that this feature is [not enabled in production](https://github.com/Automattic/wp-calypso/blob/115d846abaff846ce367a37e88ab337d3318f17c/config/production.json#L35). Should it be? Or can we remove this toggle altogether if it is not in use?